### ISSUE TITLE
Mobile Firefox typing is broken

### ISSUE THREAD

**Issue by Chiiruno** [Chiiruno/meguca#12]
https://github.com/bakape/meguca/issues/1109

**Comment by rsdr** [comment]
Just for confirmation, typing works fine on iOS devices using the latest version of Firefox, so it might be an android-only issue. 
(Then again, who cares about Apple users anyway)

**Comment by Chiiruno** [comment]
Yeah, it could be android keyboard auto correct, I'll need to test.

**Comment by rsdr** [comment]
It is the auto correct, or some kind of auto correct. One anon just tested and it looked like they were auto-correcting their words as they were being deleted, so each letter changed as they were deleted.
